ui/app/AppLayouts/Chat/views/communities/CommunityNewTokenView.qml

```diff
@@ -195,10 +195,14 @@ StatusScrollView {
             extraValidator.errorMessage: qsTr("You have used this token symbol before")
 
             onTextChanged: {
+                const cursorPos = input.edit.cursorPosition
```

Comment:
WDYT of using `font.capitalization: Font.AllUppercase` instead?

Reply:
Tried, doesn't really help here as the validators work on the real text, not the one that's displayed :)